packages/react-devtools-extensions/src/parseHookNames/parseHookNames.js

```diff
@@ -618,6 +661,22 @@ function updateLruCache(
   return Promise.resolve();
 }
 
+function extractHookMapFromSourceMap(
+  sourcemap,
+  sourceIndex: number,
+): HookMap | null {
+  let hookMap;
+  if (sourcemap.hasOwnProperty(REACT_SOURCES_EXTENSION_KEY)) {
+    hookMap = sourcemap[REACT_SOURCES_EXTENSION_KEY][sourceIndex][0];
+  } else if (sourcemap.hasOwnProperty(FB_SOURCES_EXTENSION_KEY)) {
+    hookMap = sourcemap[FB_SOURCES_EXTENSION_KEY][sourceIndex][1];
```

Comment:
Why is the hook map the second item in the fb-sources array but the first in the react-sources array? (Could use an inline comment?)